examples/README.md

```diff
@@ -0,0 +1,13 @@
+# Examples
+
+| Name     | Status         |
+| -------- | -------------- |
+| chacha20 | Lax-typechecks |
+| lob      | Typechecks     |
+| sha256   | Lax-typechecks |
+
+## How-to extract and typecheck an example
+To generate F* modules and **lax-typecheck** them for an example `<EXAMPLE>`:
+1. `cd <EXAMPLE>/proofs/fstar/extraction`
```

Comment:
The `proofs` directory doesn't exist.
Maybe we should add a script here to extract everything?

Reply:
Or just say that `make` runs the extraction as well (which is a little confusing, given that it's run from the extraction dir)